bin/uglifyjs

```diff
@@ -401,7 +401,7 @@ async.eachLimit(files, 1, function (file, cb) {
         writeNameCache("props", cache);
     })();
 
-    var SCOPE_IS_NEEDED = COMPRESS || MANGLE || BEAUTIFY || ARGS.lint;
+    var SCOPE_IS_NEEDED = true;
```

Comment:
Does it make sense to retain `SCOPE_IS_NEEDED` if it's always true?

Also: are there other locations where the same change may be needed, like `.minify()`?